_sass/layout/_search.scss

```diff
@@ -0,0 +1,31 @@
+.search__wrapper {
+  margin-top: 2 * $spacing-base;
+  margin-bottom: 4 * $spacing-base;
+}
+
+.election-checkboxes {
+  @include media($grid--small) {
+    margin-bottom: 2 * $spacing-base;
+  }
+}
+
+.searchbar {
+  margin-bottom: $spacing-base;
+
+  input {
+    height: 3rem;
+  }
+}
+
+// NOTE: These `ais` classes override the defaults from Algolia Instant Search
+.ais-Pagination {
+  margin-top: 2 * $spacing-base;
+}
+
+.ais-Hits-item {
+  width: 100% !important;
```

Comment:
@MrVantastic Why is the !important needed?

Reply:
It looks like without the `!important` added here, each hit item is no longer 100% the width of the container:
<img width="1440" alt="Screen Shot 2020-07-26 at 9 24 04 PM" src="https://user-images.githubusercontent.com/28023047/88503366-57926280-cf86-11ea-9806-dd296f32c85d.png">

We might be able to leave the `!important` there and somehow disable/ignore the error, or we can find another workaround for ensuring the width is 100%... perhaps extending the component from Algolia and overriding it more specifically that way.

Reply:
Is it true to see changes to .css I need to restart the server, unlike other changes?  This appears to be the case, which is why I did not see the difference.  It would be nice if there were other formats available from Algolia so we don't have to roll our own or make changes to theirs.

Reply:
I'm not sure, when I make changes to the .scss files, I see my server reload and I just have to manually refresh the page in the browser in order to see the changes. I haven't been running this in docker though, so it could be different, but I'm not 100% sure (I wouldn't think that css changes would need a server restart in any case).

It would be nice, but extending/expanding the Algolia components into our own shouldn't be too difficult to do. That's one of the reasons I'm so keen on getting this initial component merged in, so we can start to iterate and expand on Algolia's components to make it truly our own.

Reply:
>We might be able to leave the !important there and somehow disable/ignore the error, or we can find another workaround for ensuring the width is 100%... perhaps extending the component from Algolia and overriding it more specifically that way... extending/expanding the Algolia components into our own shouldn't be too difficult to do. That's one of the reasons I'm so keen on getting this initial component merged in, so we can start to iterate and expand on Algolia's components to make it truly our own.

@anlawyer with this in mind, would it be okay to keep the PR as-is and change the results from a grid view to a list view later by extending/overriding Algolia's component, or should we work on extending the component now?

Reply:
@MrVantastic I'd be fine with merging this one first! The only way that anyone can access this new search feature is by entering the url directly, so I think it's low-risk enough to have it merged and continue to iterate/expand in future PRs.